src/main/resources/translations/Messages_nl.properties

```diff
@@ -0,0 +1,165 @@
+plugin_loaded_players = Geladen! Gevonden spelers: %s
+plugin_migration_no_watchblock = Migratie is ingeschakeld, Maar Watch block is niet gevonden!
+plugin_migration_failed = Inporteren is gefaalt. - Kijk voor fouten!
+
+plugin_config_updated = Config upgedate naar versie: %s
+plugin_config_unknown_version = Ombekende versie: "%s", Controleer dat de config.yml wel correct is.
+
+### EVENT MESSAGES ###
+
+# Player interaction event (left-/right-click a block)
+
+event_interact_denied = &cJe mag niks doen met de block: &b%s&c, van: &b%s&c.
+event_interact_sword_not_owned = &bDeze block is van niemand.
+event_interact_sword_owned_by = &bDeze block is van: &3%s&b.
+
+# Player bucket empty event
+
+event_bucket_denied = &cJe mag hier geen emmers legen van: &b%s&c.
+
+# Block place event
+
+event_block_place_denied = &cJe mag geen blockken plaatsen tegen de blockken van: &b%s&c.
+
+# Block break event
+
+event_block_break_denied = &cJe mag deze blockken niet breken van: &b%s&c.
+
+### COMMANDS ###
+
+# Disown command
+disown_command_disowning_blocks = &9Verwijderen eigenaarschap van: &c%s&9. Dit kan even duren.
+disown_command_usage = &9Gebruik: &b/&3%s &2<Gebruikersnaam>
+disown_thread_complete = &c%s Blockken&9 Hebben geen eigenaarschap meer.
+
+# Disown world command
+disownworld_command_disowning_blocks = &9Verwijderen eigenaarschappen blockken van wereld &c%s&9. Dit kan even duren.
+disownworld_command_usage = &9Gebruik: &b/&3%s &2<Wereld>
+disownworld_thread_complete = &c%s Blockken&9 Hebben gen eigenaarschappen meer.
+
+# Friend command
+friend_command_console_already_friends = &b%s&c Is al vrienden met: &b%s
+friend_command_console_now_friends = &b%s&a Is nu vrienden met: &b%s
+friend_command_console_usage = &9Gebruik: &b/&3%s &2<gebruiekrsnaam> <Vriend>
+
+friend_command_already_friends = &cJe bent al vrienden met &b%s
+friend_command_now_friends = &aJe bent nu vrienden met &b%s
+friend_command_usage = &9Gebruik: &b/&3%s &2<Gebruik>
+
+# Friends (listing) command
+
+friends_command_console_usage = &9Gebruik: &b/&3%s &2<Gebruikersnaam>
+
+# The following three entries are used to lay out the friends list as displayed to the user. For example, if we
+# set list_header to "== &d[&6F&d]&f ==", list_row to "&6> %s" and list_separator to " &3| &b", our list will look
+# something like this:
+#
+# == &d[&6F&d]&f ==
+# &6> <player> &3| &b<player> &3| &b<player>
+# &6> <player> &3| &b<player> &3| &b<player>
+# &6> <player> &3| &b<player> &3| &b<player>
+#
+# For the most parts, translators won't need to touch this, aside from translating "Friends"
+
+friends_command_list_header = == &d[&6Vrienden&d]&f ==
+friends_command_list_row = &b%s
+friends_command_list_separator = &3, &b
+
+friends_command_no_friends = &cGeen vrienden gevonden :(
+
+# Setowner command
+
+setowner_command_changing_ownership = &9Veranderen eigenaarschappen van: &c%s&9. Dit kan even duren.
+setowner_command_make_worldedit_selection = &cMaak een geldige WorldEdit cuboid selectie.
```

Comment:
`cuboid` should be translatable.. But I don't know if WorldEdit actually translates it.